docs/en/observability/observability-introduction.asciidoc

```diff
@@ -2,77 +2,115 @@
 
-Install and configure {heartbeat} on your servers to monitor host availability, service
-uptime, web site endpoints, and API monitoring. For detailed
-monitor summaries, with support for monitors from multiple locations, drill-down into
-the {uptime-app}.
+For more information, see <<analyze-metrics,Infrastructure Monitoring>>.
 
 [float]
 [[user-experience-overview]]
-== User experience data
+== Real user monitoring (RUM)
+
+Quantify and analyze the perceived performance of your web application with {user-experience} data, powered by the APM RUM agent. Unlike testing environments, {user-experience} data reflects real-world user experiences.
+
+On the {observability} **Overview** page, the *{user-experience}* chart provides a snapshot of core web vitals for the service with the most traffic.
+
+[role="screenshot"]
+image::images/obs-overview-ue.png[Summary of {user-experience} metrics on the {observability} overview page]
 
-{user-experience} data, powered by the APM Real User Monitoring (RUM) agent,
-provides a way to quantify and analyze the perceived performance of your web application.
+You can then drill down into the {user-experience} dashboard by clicking *Show dashboard* too see data by URL, operating system, browser, and location.
+
+For more information, see <<user-experience,{user-experience}>>.
+
+[float]
+[[logs-overview]]
+== Log monitoring
+
+Analyze log data from your hosts, services, Kubernetes, Apache, and many more.
+
+On the {observability} **Overview** page, the *Log Events* chart helps you detect and inspect possible log anomalies across each of your ingested log sources to determine if the log rate is outside of your expected bounds.
+
+[role="screenshot"]
+image::images/log-rate.png[Summary of Log Events on the {observability} overview page]
+
+You can then drill-down into the {logs-app} by clicking *Show log stream* to view a live stream of your logs and filter, pin, or highlight the data you need.
+
+For more information, see <<monitor-logs,Log monitoring>>.
 
 [float]
 [[synthetic-monitoring-overview]]
-== Synthetic monitoring data
+== Synthetic monitoring
 
-beta[] Synthetic monitoring allows you to simulate actions and requests that an end-user would perform
-on your site at predefined intervals and in a controlled environment.
+beta[] Simulate actions and requests that an end-user would perform on your site at predefined intervals and in a controlled environment.
 The end result is rich, consistent, and repeatable data that you can trend and alert on.
 
+For more information, see <<monitor-uptime-synthetics,Synthetic monitoring>>.
+
+[float]
+[[universal-profiling-overview]]
+== Universal Profiling
+beta[] Build stack traces to get visibility into your system without application source code changes or instrumentation. Use flamegraphs to explore your systems performance and identify the most expensive lines of code,  increase CPU resource efficiency, debug performance regressions, and reduce cloud spend.
```

Comment:
```suggestion
beta[] Build stack traces to get visibility into your system without application source code changes or instrumentation. Use flamegraphs to explore system performance and identify the most expensive lines of code, increase CPU resource efficiency, debug performance regressions, and reduce cloud spend.
```
That way you avoid having to make this possessive or worry about singular/plural.

Reply:
Much better, especially when I seem to have forgotten how to make things possessive.